PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.azureexplorer/src/com/microsoft/azuretools/azureexplorer/forms/AddNewClusterForm.java

```diff
@@ -65,6 +73,19 @@ protected void configureShell(Shell newShell) {
         newShell.setText("Link New HDInsight Cluster");
     }
 
+    private void refreshContainers(@NotNull ClientStorageAccount storageAccount) {
+    	try {
+        	containersComboBox.removeAll();
+
+			StorageClientSDKManager.getManager().getBlobContainers(storageAccount.getConnectionString())
+					.forEach(blob -> containersComboBox.add(blob.getName()));
+			
+			// not find setMaximumRowCount(*) method in SWT
```

Comment:
Just make sure that the combobox UI is working

Reply:
Combobox UI does work in my tests.